gymnasium_robotics/envs/maze/maze.py

```diff
@@ -274,9 +274,9 @@ def add_xy_position_noise(self, xy_pos: np.ndarray) -> np.ndarray:
     def compute_reward(
         self, achieved_goal: np.ndarray, desired_goal: np.ndarray, info
     ) -> float:
-        distance = np.linalg.norm(achieved_goal - desired_goal, axis=-1)
+        distance = np.linalg.norm(achieved_goal - desired_goal, ord = 2, axis=-1)
```

Comment:
Why was `ord=2,` added? That is the default behavior anyway

Reply:
Correct, wanted to put it out explicitly.